nncf/config/extractors.py

```diff
@@ -73,15 +73,15 @@ def extract_algo_specific_config(config: NNCFConfig, algo_name_to_match: str) ->
     return next(iter(matches))
 
 
-def extract_range_init_params(config: NNCFConfig) -> Optional[Dict[str, object]]:
+def extract_range_init_params(config: NNCFConfig, name: str) -> Optional[Dict[str, object]]:
     """
     Extracts parameters of the quantization range initialization algorithm from the
     compression algorithm NNCFconfig.
 
     :param config: An instance of the NNCFConfig.
     :return: Parameters of the quantization range initialization algorithm.
```

Comment:
Please update docstring according to the introduced `name` argument.

Reply:
I added the docstring for the `algorithm_name` parameter.